includes/model/Openedx_Woocommerce_Plugin_Enrollment.php

```diff
@@ -219,8 +223,19 @@ public function save_enrollment( $post, $enrollment_arr, $enrollment_action ) {
 
         // Get the old post metadata.
         $old_course_id           = get_post_meta($post_id, 'course_id', true);
-        $old_email            = get_post_meta($post_id, 'email', true);
+        $old_email               = get_post_meta($post_id, 'email', true);
         $old_mode                = get_post_meta($post_id, 'mode', true);
+        $old_request_type        = get_post_meta($post_id, 'is_active', true);
```

Comment:
Why is the old_request_type associated with 'is_active'? Is that the name of the variable?

Reply:
Yes, "is_active" is the name of the variable that defines if it is an "enroll" or "unenroll"

Reply:
Can we change that to a more descriptive variable like request_type?

Reply:
Oh! I know why we call it is_active because when we request to the API, we will send is_active=False if the request type is unenrollment, but I still think that calling that variable is_active is not the best way. What do you think?

Reply:
I just adjusted it, I've added the field to store the value of the request type and access it without any issues or additional conditions.